ecommerce-frontend/src/user/Signin.js

```diff
@@ -91,6 +94,13 @@ const SignIn = () => {
 
   const redirectUser = () => {
     if (redirectToReferrer) {
+      if (user && user.role === 1) {
+        return <Redirect to="/admin/dashboard" />;
+      } else {
+        return <Redirect to="/user/dashboard" />;
```

Comment:
Create a `utils` folder and create a constant name `ROUTES` and place all these static routes there. So there is a consistency across the app.